mon-pix/tests/unit/routes/authentication/login-oidc-test.js

```diff
@@ -95,30 +95,59 @@ module('Unit | Route | login-oidc', function (hooks) {
           assert.strictEqual(sessionStub.data.nextURL, '/organisations/PIXOIDC01/acces');
         });
 
-        test('should build the url from the intent name and contexts in session data nextUrl', async function (assert) {
-          // given
-          const authenticateStub = sinon.stub().resolves();
-          const sessionStub = Service.create({
-            attemptedTransition: { intent: { name: 'organizations.access', contexts: ['PIXOIDC01'] } },
-            authenticate: authenticateStub,
-            data: {},
+        module('when there is at least one context', function () {
```

Comment:
Ici il faudrait 3 when distincts : 
1. `'when TransitionIntent is a URLTransitionIntent'`
2. `'when TransitionIntent is a NamedTransitionIntent with at least one route context'`
3. `'when TransitionIntent is a NamedTransitionIntent with no route context'`